draft-bormann-jsonpath-iregexp.md

```diff
@@ -85,7 +83,7 @@ called regular expressions in programming.
 in this specification; the plural is "I-Regexps".
 
 I-Regexp does not provide advanced regexp features such as capture groups, lookahead, or backreferences.
-It supports only a Boolean matching capability, i.e. testing whether a given regexp matches a given piece of text.
+It supports only a Boolean matching capability, i.e., testing whether a given regexp matches a given piece of text.
```

Comment:
comma is un-idiomatic

Reply:
Well, the RFC editor will put it in anyway (CMOS requires it).

Re non-ASCII Unicode:
So far, ASCII is only mentioned in conjunction with the intentions some regexp writers have when writing \d, w, which we don't support.
So I hope there is no ASCII-related blunder in the document; everything is Unicode.

4 hours 50 minutes left...

Reply:
Having read the draft (getting pretty good!) I'd really like to be a little more militant: implementations MUST NOT limit themselves to ASCII characters and MUST support the Unicode character property set. Is that reasonable?

Reply:
Not sure that I completely agree with what I wrote in a228258, but this is at least the right direction.